docs/docs/guides/web3_migration_guide/providers_migration_guide.md

```diff
@@ -245,7 +245,7 @@ You can use it by installing `web3-providers-ipc` and creating a new instance. S
 you can pass it on to the Web3 instance.
 
 ```ts
-import IpcProvider from 'web3-providers-ipc';
+import { IpcProvider } from 'web3-providers-ipc';
```

Comment:
What's the reason for this change? Our other providers (http and ws) import via the default, and [IpcProvider](https://github.com/web3/web3.js/blob/3b95b5ef52fcda4c3e40066a2cf54f9264389fab/packages/web3-providers-ipc/src/index.ts#L62) is still the default export